### PR TITLE
Sensitivity on the active power setpoint of an HVDC line

### PR DESCRIPTION
**Pull request by Djazouli**
Signed-off-by: Gael Macherel <gael.macherel@artelys.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by Djazouli** [comment]
Does not work in AC currently, I'm trying to figure out why

File: src/test/java/com/powsybl/openloadflow/sensi/dc/DcSensitivityAnalysisContingenciesTest.java [hunk 1]
```diff
@@ -883,6 +885,87 @@ void testRemainingGlskFactors() {
         assertEquals(Double.NaN, getContingencyValue(result, "l34", "glsk", "l56"), LoadFlowAssert.DELTA_POWER);
     }
 
+    @Test
+    void testHvdcSensiRescale() {
+        double sensiChange = 10e-4;
+        // test injection increase on loads
+        Network network = HvdcNetworkFactory.createNetworkWithGenerators();
+        network.getGeneratorStream().forEach(gen -> gen.setMaxP(2 * gen.getMaxP()));
```
Comment (annetill) [inline src/test/java/com/powsybl/openloadflow/sensi/dc/DcSensitivityAnalysisContingenciesTest.java:893]: Why do you need this line on generator max P?

File: src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisContingenciesTest.java [hunk 2]
```diff
@@ -428,7 +429,57 @@ void testBusVoltagePerTargetVFunctionRef() {
...
+                line -> (network1.getLine(line).getTerminal1().getP() - network2.getLine(line).getTerminal1().getP()) / sensiChange
+            ));
+
+        HvdcWriter hvdcWriter = HvdcWriter.create();
+        SensitivityFactorReader reader = createHvdcReader(variableAndFunction);
+        sensiParameters.getLoadFlowParameters().getExtension(OpenLoadFlowParameters.class).setSlackBusSelector(new NameSlackBusSelector("b1_vl_0")); // the most meshed bus selected in the loadflow
+        sensiProvider.run(network, VariantManagerConstants.INITIAL_VARIANT_ID, Collections.singletonList(new Contingency("l25", new BranchContingency("l25"))),
+            sensiParameters, reader, hvdcWriter);
+
+        // FIXME
```
Comment (annetill) [inline src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisContingenciesTest.java:475]: I have remplaced LCCs by VSCs but it still does not work with distributed slack.

**Comment by annetill** [comment]
@geofjamg I have after merge two unit tests that failed. The error is not obvious, so if you have any idea, it would be really helpful. Thanks a lot!

**Review by geofjamg — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Pull request merged**